README.md

```diff
@@ -29,3 +46,16 @@ const customTheme = createMuiTheme({
     <SchemaViewer />
 </ThemeProvider>
 ```
+
+## Contributing
+This repository uses Neutrino and `neutrino-preset-react-components` for developing, previewing, and building React components. To get started:
```

Comment:
`neutrino-preset-react-components` is not used in this repo. We can omit this:

This repository uses Neutrino for developing, and building React components. To get started:

Reply:
Applied changes!